app/subscriber/src/features/my-reports/hooks/useColumns.tsx

```diff
@@ -51,11 +51,16 @@ export const useColumns = (
       width: '100px',
       cell: (cell) => (
         <Row gap="0.5rem">
-          <FaEye
-            className="btn-link"
-            onClick={() => navigate(`/report/instances/${cell.original.id}/view`)}
-            title="Instances"
-          />
+          <Show visible={!!cell.original.instances.length}>
+            <FaEye
+              className="btn-link"
+              onClick={() => {
+                const instances = cell.original.instances;
+                navigate(`/report/instances/${instances[instances.length - 1].id}/view`);
```

Comment:
Does this make sense to get the latest report instance?

Reply:
I would remove this column for now.  It causes too much confusion.  The report instance page you were building was only meant as a page to view reports that have been sent out by email.

Reply:
Column removed